<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QssDialog</class>
 <widget class="QDialog" name="QssDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QSS Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="QssHorizontalLayout">
     <item>
      <widget class="QssTextEdit" name="QssEdit"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="QssVerticalLayout">
       <item>
        <widget class="QPushButton" name="QssLoadButton">
         <property name="text">
          <string>Load...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="QssSaveButton">
         <property name="text">
          <string>Save...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="QssSaveDefaultButton">
         <property name="text">
          <string>Save as default</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="QssBasicButton">
         <property name="text">
          <string>Basic</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="QssMediumButton">
         <property name="text">
          <string>Medium</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="QssAdvancedButton">
         <property name="text">
          <string>Advanced</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="QssVerticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="QssButtonBoxHorizontalLayout">
     <item>
      <widget class="QLabel" name="QssValidityLabel">
       <property name="text">
        <string>Valid Qss</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="QssButtonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QssTextEdit</class>
   <extends>QTextEdit</extends>
   <header>QssTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>QssButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>QssDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>QssButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>QssDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
